repository-data/webfiles/src/main/resources/site/src/scss/components/_article-section-nav.scss

```diff
@@ -1,28 +1,40 @@
 @import "../units/list";
 
-/* Sticky nav */
-#sticky-nav {
-    a.active,
-    span.active {
-        border-left: $baseline-grid-unit solid #ffb81c;
-        padding-left: 12px;
-        color: $col-20;
-        text-decoration: none;
-    }
+/* -- sticky nav rewrite 2020 start -- */
+
+.navMarker{
+    //border-left: $baseline-grid-unit solid #ffb81c;
+    border-left: solid #ffb81c;
+    padding-left: 12px;
+    //color: $col-20;
+    text-decoration: none;
 }
 
-.article-section-nav-outer-wrapper {
-
-}
+/* Sticky nav */
+//#sticky-nav {
+//    a.active,
+//    span.active {
+//        border-left: $baseline-grid-unit solid #ffb81c;
+//        padding-left: 12px;
+//        color: $col-20;
+//        text-decoration: none;
+//    }
+//}
+
+//.article-section-nav-outer-wrapper {
+//
+//}
```

Comment:
It's better not to leave commented out code.

Reply:
I agree, but I wanted to leave it in, until you reviewed it and felt that it could be deleted fully. I have now removed this.